formal/icache/Makefile

```diff
@@ -0,0 +1,16 @@
+# A simple wrapper around fusesoc to make it a bit easier to run the formal flow
+
+core-name   := lowrisc:fpv:ibex_icache_fpv
+vlnv        := $(subst :,_,$(core-name))
+build-root  := $(abspath ../../build/$(vlnv))
```

Comment:
Is that really a problem? They are pretty standard in Makefiles, and I think they're easier to read (for the same reason that we have --compound-arguments, rather than --compound_arguments).

Reply:
No, not a problem at all; no need to change things.